.github/workflows/build.yml

```
@@ -17,9 +17,14 @@ on:
   push:
     branches:
       - 2022 #TODO: Switch branch when strategy is finalized
+    tags:
```

```suggestion
    tags:
    branches:
      - 2022 #TODO: Switch branch when strategy is finalized
```

thoughts on this order change? just looks a little cleaner to me